website/docs/help-and-support/troubleshooting-guide/deployment-errors.mdx

```diff
@@ -135,6 +135,59 @@ As the error message in the server logs suggests, your Appsmith instance has mis
 
 #### Solution
 
-To resolve this error, you must [upgrade to the checkpoint version](/getting-started/setup/instance-management#checkpoint-upgrade), in this case, `v1.9.2`.
+To resolve this error, you must [upgrade to the checkpoint version](/getting-started/setup/instance-management/update-appsmith#upgrade-to-a-checkpoint-version), in this case, `v1.9.2`.
 
-To prevent similar issues in the future, consider [enabling auto updates](/getting-started/setup/installation-guides/docker#enabling-appsmith-auto-updates) for your Appsmith instance. If you already had auto updates on and still encountered this issue, please contact the [Appsmith team](https://discord.com/invite/rBTTVJp) to determine the cause of the deployment problem
\ No newline at end of file
+To prevent similar issues in the future, consider [enabling auto updates](/getting-started/setup/installation-guides/docker#enabling-appsmith-auto-updates) for your Appsmith instance. If you already had auto updates on and still encountered this issue, please contact the [Appsmith team](https://discord.com/invite/rBTTVJp) to determine the cause of the deployment problem
+
+## Illegal instruction (core dumped)
+MongoDB fails to start, after migrating to MongoDB version 5.
+
+#### Error message
+
+<Message
+messageContainerClassName="error" 
+messageContent="Illegal instruction (core dumped) mongod –fork –port 27017 –dbpath “$MONGO_DB_PATH"></Message>
+
+#### Cause
+MongoDB version 5 has specific CPU requirements. If these requirements are not met the server wouldn't start.
+
+#### Solution
+To resolve this error, you must upgrade your infrastructure. See the [List of complaint hardware](https://www.mongodb.com/docs/manual/administration/production-notes/#platform-support).
+
+If you can't upgrade your infrastructure or can't upgrade your embedded Appsmith MongoDB version:
+* You may use an external MongoDB version 5 or higher on [MongoDB Atlas](https://www.mongodb.com/pricing).
+* **Or** downgrade your Appsmith instance to v1.8.15. Follow the steps below to downgrade to v1.8.15:
+
+  1. Access your Appsmith instance and make the below change to the `docker-compose.yml` file:
+
+  <Tabs groupId="appsmithEditions" queryString="current-edition">
+    <TabItem value="CE" label="Community Edition" default>
+
+    ```yaml
+    services:
+    appsmith:
+      #highlight-next-line
+      image: index.docker.io/appsmith/appsmith-ce:v1.8.15
+      container_name: appsmith
+    ```
+    
+    </TabItem>
+    <TabItem value="BE" label="Business Edition">
+
+    ```yaml
+    services:
+    appsmith:
+       #highlight-next-line
+      image: index.docker.io/appsmith/appsmith-ee:v1.8.15
+      container_name: appsmith
+    ```
+    </TabItem>
+  </Tabs>
+
+
+  2. Run the restart command: 
+
+  ```bash
+  docker-compose up -d
+  ```
+Your server starts with version 1.8.15, which makes all the necessary schema changes to your Appsmith instance. 
```

Comment:
Should we also call out that v1.8.15 misses out on all the latest features, and should only be seen as a temporary solution until the usersare able to provision a VM that supports Mongo v5 or provision an external MongoDB?

Reply:
v1.8.15 looks like a very bad path on the long term and should only be used to unblock user on the short term.